src/test/java/org/mockitousage/matchers/CapturingArgumentsTest.java

```diff
@@ -315,4 +315,31 @@ public void captures_correctly_when_captor_used_on_pure_vararg_method() throws E
         verify(mock).varargs(eq(42), argumentCaptor.capture());
         Assertions.assertThat(argumentCaptor.getValue()).contains("capturedValue");
     }
+
+    @Test
+    public void should_capture_null_single_argument_in_vararg_method() {
+        // given
+        ArgumentCaptor<String> argumentCaptor = ArgumentCaptor.forClass(String.class);
+
+        // when
+        mock.varargs(null);
+
+        // then
+        verify(mock).varargs(argumentCaptor.capture());
+        Assertions.assertThat(argumentCaptor.getValue()).isNull();
+    }
+
+    @Test
+    public void should_capture_null_array_in_method_vararg() {
+        // given
+        ArgumentCaptor<String[]> argumentCaptor = ArgumentCaptor.forClass(String[].class);
+
+        // when
+        String[] nullArray = null;
+        mock.varargs(nullArray);
+
+        // then
+        verify(mock).varargs(argumentCaptor.capture());
+        Assertions.assertThat(argumentCaptor.getValue()).isNull();
```

Comment:
@ChristianSchwarz not sure about that, because 'null' has been passed to method, hasn't it ?

If 'noting should be captured' then what getValue method should return in that case?